app/deed/search/routes.py

```diff
@@ -12,7 +11,10 @@ def search_deed():
     def redirect_to_view_deed():
         deed_ref_num = request.form['deedRefNumber']
         url = url_for('deed.view.view_deed', deedRefNum=deed_ref_num)
-        return redirect(url, code=303)
```

Comment:
Cool, only reason is because 302 is title "found" in http and we are in the process of finding that I thought it would be.